app/assess/templates/macros/scores_justification.html

```diff
@@ -4,7 +4,7 @@
 
 {% macro scores_justification(scores_submitted, form, score_list, latest_score, application_id, sub_criteria_id, COF_score_list, score_error, justification_error) %}
 
-<div class="govuk-grid-column-two-thirds s26-scoring">
+<div class="govuk-grid-column-full s26-scoring">
```

Comment:
`s26-scoring` could be removed here/re-named since it's relevant to the design sprint